<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>411</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>622</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>622</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Changer mot de passe</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="password_box">
     <property name="locale">
      <locale language="French" country="France"/>
     </property>
     <property name="title">
      <string>Mot de passe</string>
     </property>
     <layout class="QVBoxLayout" name="password_layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rules_box">
     <property name="title">
      <string>Règles</string>
     </property>
     <layout class="QVBoxLayout" name="rules_layout">
      <item>
       <widget class="QScrollArea" name="rules_scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="rulesAreaContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>582</width>
           <height>195</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="ruleArea_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ruleButtons_layout">
        <item>
         <widget class="QPushButton" name="addRule_button">
          <property name="text">
           <string>Ajouter règle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteRules_button">
          <property name="text">
           <string>Supprimer règles</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="changeMasterkey_box">
     <property name="title">
      <string>Changer clé maître</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="changeMasterkey_layout">
      <item>
       <widget class="QLineEdit" name="masterkey_lineEdit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="drawMasterkey_checkbox">
        <property name="text">
         <string>Afficher clé maître</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
